### DISCUSSION TITLE
Feature request: Week picker

### DISCUSSION THREAD

**Discussion by atharva-upadhye** [gpbl/react-day-picker#2111] [Ideas]
Can we have a week picker as well?

UI something like:
![image](https://github.com/gpbl/react-day-picker/assets/70993547/c6ea917a-4d93-428a-b33a-7fb38720279e)

**Comment by gpbl** [comment]
Hi @atharva-upadhye! You can see an example of selecting a week here:

https://react-day-picker.js.org/advanced-guides/custom-selections#week-selection

![2024-05-15 5 53 50 PM](https://github.com/gpbl/react-day-picker/assets/120693/1962a213-01c0-40cb-a3b4-0cb93e8beb14)
